src/common/cockpittemplate.c

```diff
@@ -140,3 +141,19 @@ cockpit_template_expand (GBytes *input,
 
   return g_list_reverse (output);
 }
+
+GList *
+cockpit_template_expand (GBytes *input,
```

Comment:
It seems that this functionality in common should be generic, and this function should accept the start_marker and end_marker fields. The callers should need to specify them.